src/locales/en/translation.json

```diff
@@ -368,6 +368,10 @@
         "header": "Enter your password",
         "paragraph": "We need to make sure it’s you before you can manage how you get security codes."
       },
+      "globalLogout": {
+        "header": "Enter your password",
+        "paragraph": "We need to make sure it’s you before you can manage how you get security codes."
```

Comment:
Security codes? Does this content need to be updated something like "We need to make sure it’s you before you can sign out of all devices?